samples/boards/stm32/power_mgmt/blinky/README.rst

```diff
@@ -0,0 +1,42 @@
+.. _stm32-pm-blinky-sample:
+
+Blinky
+######
+
+Overview
+********
+
+Blinky is a simple application which blinks an LED forever using the :ref:`GPIO
+API <gpio_api>` in low power context.
+
+.. _stm32-pm-blinky-sample-requirements:
+
+Requirements
+************
+
+You will see this error if you try to build Blinky for an unsupported board:
+
+.. code-block:: none
+
+   Unsupported board: led0 devicetree alias is not defined
+
+The board must have an LED connected via a GPIO pin. These are called "User
```

Comment:
"a LED"

Reply:
I think we say "ell ee dee", so "an ell ee dee" would be correct

Reply:
Yes, `an LED` is correct.  https://blog.apastyle.org/apastyle/2012/04/using-a-or-an-with-acronyms-and-abbreviations.html

Reply:
Ok, Sorry.